engine/battle/animations.asm

```diff
@@ -176,8 +176,13 @@ PlayAnimation:
 	ld h, [hl]
 	ld l, a
 .animationLoop
+	vc_hook FPA_006_End
+	vc_blue_hook FPA_007_End
 	ld a, [hli]
+	vc_red_hook FPA_007_End
+	vc_blue_hook FPA_009_End
```

Comment:
Since these are meant to be the same hook in both versions, just doing different things, how about this: put a single `vc_hook foobar` here, and then in the Red template, you have `[FPA 007 End@foobar]`, and Blue has `[FPA 009 End@foobar]`. Ideally this sort of alias usage will allow getting rid of the red and blue specific macros.